cfn-resources/encryption-at-rest/test/encryptionatrest.create-sample-cfn-request.sh

```diff
@@ -18,9 +18,7 @@ if [ "$#" -ne 1 ]; then usage; fi
 if [[ "$*" == help ]]; then usage; fi
 
 projectID="${1}"
-jq --arg pubkey "$ATLAS_PUBLIC_KEY" \
-	--arg pvtkey "$ATLAS_PRIVATE_KEY" \
-	--arg projectID "$projectID" \
+jq --arg projectID "$projectID" \
```

Comment:
.desiredResourceState.ApiKeys.PublicKey?|=$pubkey | .desiredResourceState.ApiKeys.PrivateKey?|=$pvtkey |

Not required, it may fail if place holders are removed

Reply:
I updated `encryptionatrest.create-sample-cfn-request.sh `: removed the apiKeys and added the profile